src/pages/CrossChain/index.tsx

```diff
@@ -57,10 +57,12 @@ function CrossChain({ visible }: { visible: boolean }) {
         const { chains = [], tokens = [] } = squid
         const chainSupports = chains.map(e => Number(e.chainId)).filter(id => SUPPORTED_NETWORKS.includes(id))
         const formattedTokens: WrappedTokenInfo[] = []
-        tokens.forEach(token => {
-          if (typeof token.chainId === 'string' || !chainSupports.includes(token.chainId)) return
-          formattedTokens.push(new WrappedTokenInfo(token as TokenInfo))
+        tokens.forEach((token: any) => {
+          token.chainId = +token.chainId
```

Comment:
I noticed that `token.chainId` is already of type number or string
Do we need to change the type of token to `any` in this case?

Reply:
still typescript error
I update by copy to a new token. don't use any 
<img width="475" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/33005392/a2eb954c-663b-4f01-b04f-c653cef722fa">

Reply:
Nice!